plugins/mac-capture/mac-sck-video-capture.m

```diff
@@ -160,6 +160,10 @@ static bool init_screen_stream(struct screen_capture *sc)
 
                 [sc->stream_properties setWidth:(size_t) target_window.frame.size.width];
                 [sc->stream_properties setHeight:(size_t) target_window.frame.size.height];
+
+                if (@available(macOS 14.2, *)) {
+                    [sc->stream_properties setIncludeChildWindows:YES];
+                }
```

Comment:
According to the header this should be enabled by default, is that not actually the case?

Reply:
True, but I'd rather have it explicitly enabled because that's what users expect right now and avoiding a similar situations where macOS' internal defaults change.